src/status_im/config.cljs

```diff
@@ -133,3 +135,11 @@
 (def community-accounts-selection-enabled? true)
 (def fetch-messages-enabled? (enabled? (get-config :FETCH_MESSAGES_ENABLED "1")))
 (def test-networks-enabled? (enabled? (get-config :TEST_NETWORKS_ENABLED "0")))
+
+(def default-wallet-connect-metadata
+  {:name "Status"
+   :description
+   "Status is a secure messaging app, crypto wallet, and Web3 browser built with state of the art technology."
+   :url "https://status.app"
+   :icons
+   ["https://res.cloudinary.com/dhgck7ebz/image/upload/f_auto,c_limit,w_1080,q_auto/Brand/Logo%20Section/Mark/Mark_01"]})
```

Comment:
the formatting looks weird

Reply:
I extracted it from our website just to provide some values, but we can update it later on

Reply:
> Not a change, but is it possible to pass icon from disk rather than a link? What if the infra team stops using Cloudinary tomorrow ?

From all examples I've seen, only urls are passed for icons. I think we could do some research and update this part in a follow up in case it is needed. Also copies should be temporal, we should get better ones from the marketing or design team ultimately.